package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./backend/bin/www",
-    "back": "createdb endeavor_db && cd backend && psql -f db/seed.sql -d endeavor_db && npm run dev",
+    "back_first": "createdb endeavor_db && cd backend && psql -f db/seed.sql -d endeavor_db && npm run dev",
```

Comment:
still playing with the name? back_firsttime or init_back / back_init? or back_first is fine. i'll approve.
thanks for the change

Reply:
Which one do you feel is more relevant?